packages/kit/src/views/Send/pages/SendConfirm/TxAdvancedSettingsContainer.tsx

```
@@ -0,0 +1,336 @@
+    async () =>
+      backgroundApiProxy.serviceNetwork.getVaultSettings({ networkId }),
+    [networkId],
+  ).result;
+
+  const dataContent = useMemo(() => {
+    if (!unsignedTxs || unsignedTxs.length === 0) {
+      return '';
+    }
+    return unsignedTxs.reduce((acc, unsignedTx) => {
+      const tx = unsignedTx.encodedTx as IEncodedTxEvm;
+      if (tx && tx.data) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)